src/GafferScene/Camera.cpp

```diff
@@ -57,8 +61,30 @@ Camera::Camera( const std::string &name )
 {
 	storeIndexOfNextChild( g_firstPlugIndex );
 	addChild( new StringPlug( "projection", Plug::In, "perspective" ) );
+	addChild( new StringPlug( "controlMode", Plug::In, "fieldOfView" ) );
```

Comment:
Last I talked to Murray, he felt it was reasonable to keep fieldOfView as the default.  FieldOfView is a bit more convenient for people who don't think a lot about cameras, or are coming from a videogame background where FOV is common.  People who want a focalLength/aperture know what they want, and can probably manage to flip the one toggle ( focal length mode is always more work to set up anyway, because you need to think about what aperture of camera you're working with ).

Reply:
Happy to defer to Murray on the default, especially since we can change it at a later date via userDefaults. But how about the backwards compatibility?

Reply:
I do still need to do testing of backwards compatibility.  But if we're defaulting to fieldOfView perspectiveMode, then that much should be compatible by default?

Reply:
Haven't tested this systematically yet, but the main compatibility issue I've happened across so far is that the old screen window fit code is equivalent to filmFit = Fit, but the default we want is filmFit = Horizontal - so I'll have to do a compatibility config that sets a render setting override on all cameras loaded from old scenes.

Reply:
I guess the other issue is where folks have explicitly put a screenWindow parameter on manually  using a Parameters node. Perhaps we can punt on that though, since they were working around the problems we're fixing here? Might it be worth a word on the dev list?